packages/server/lib/controllers/connection.controller.ts

```diff
@@ -706,6 +708,21 @@ class ConnectionController {
             next(err);
         }
     }
+    public async getPossibleFilters(req: Request, res: Response, next: NextFunction) {
+        try {
+            const { success: sessionSuccess, error: sessionError, response } = await getUserAccountAndEnvironmentFromSession(req);
+            if (!sessionSuccess || response === null) {
+                errorManager.errResFromNangoErr(res, sessionError);
+                return;
+            }
+            const { environment } = response;
+
+            const integrations = await configService.getAllNames(environment.id);
```

Comment:
This would return all integrations, even the one that have no connection.  I feel we should only show integrations that have at least one connection. @bastienbeurier what do you think?

Reply:
100% that's a good call

Reply:
@hassan254-prog do you mind taking care of only showing integrations that have at least one connection in the drop down list. Thank you very much.